src/iterative_ensemble_smoother/_iterative_ensemble_smoother.py

```diff
@@ -199,32 +178,33 @@ def update(self, param_ensemble: npt.NDArray[np.double]) -> npt.NDArray[np.doubl
     def __repr__(self) -> str:
         return (
             f"SIES(ensemble_size={self._initial_ensemble_size}, "
-            f"max_steplength={self.max_steplength}, min_steplength={self.min_steplength}, "
-            f"dec_steplength={self.dec_steplength})"
+            f"steplength_schedule={self.steplength_schedule}, seed={self.seed})"
```

Comment:
Should we have optional input parameters in __repr__?

Reply:
Good question. If we want the `eval(repr(instance)) == instance` property to hold, then we do. But it will not hold when we pass in a function anyway. So maybe wanting this property to hold is overkill.